pkg/istio-agent/grpcxds/grpc_bootstrap.go

```diff
@@ -51,17 +52,17 @@ type XdsServer struct {
 }
 
 type CertificateProvider struct {
-	Name   string      `json:"name,omitempty"`
-	Config interface{} `json:"config,omitempty"`
+	PluginName string      `json:"plugin_name,omitempty"`
+	Config     interface{} `json:"config,omitempty"`
 }
 
 const FileWatcherCertProviderName = "file_watcher"
 
 type FileWatcherCertProviderConfig struct {
-	CertificateFile   string               `json:"certificate_file,omitempty"`
-	PrivateKeyFile    string               `json:"private_key_file,omitempty"`
-	CACertificateFile string               `json:"ca_certificate_file,omitempty"`
-	RefreshDuration   *durationpb.Duration `json:"refresh_interval,omitempty"`
+	CertificateFile   string          `json:"certificate_file,omitempty"`
+	PrivateKeyFile    string          `json:"private_key_file,omitempty"`
+	CACertificateFile string          `json:"ca_certificate_file,omitempty"`
+	RefreshDuration   json.RawMessage `json:"refresh_interval,omitempty"`
```

Comment:
does just time.Duration work?

Reply:
Possibly? I rewrote this to match their unmarshaling

Reply:
I think it would but matching there's is better